packages/compass-aggregations/src/components/pipeline-builder-workspace/pipeline-as-text-workspace/pipeline-preview.tsx

```diff
@@ -38,7 +42,10 @@ const centerStyles = css({
 const messageStyles = css({ marginTop: spacing[3] });
 
 const documentListStyles = css({
-  overflow: 'scroll',
+  overflow: 'auto',
+  '.document-list': {
```

Comment:
Maybe let's allow to pass className to the Document list if this is required?

Reply:
I removed this code to align document list with the PipelineOutputMenu when we have scrollbar. Not sure if we need to pass the class prop to DocumentList now